tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "rootDir": "./",
+    "outDir": "dist/tmp",
+    "noEmitOnError": true,
+    "sourceMap": true,
+    "module": "amd",
+    "target": "es5",
+    "types": [
+      "mendix-client",
+      "jasmine"
+    ],
+    "moduleResolution": "node",
+    "declaration": false,
+    "removeComments": true,
+    "noLib": false,
+    "watch": true,
```

Comment:
When the watch is set to true, the `> npm test` will not work. Set it to false. Like run it continuously use `> tsc -w`